### ISSUE TITLE
Undo/redo of large tempo adjustment is no longer possible

### ISSUE THREAD

**Issue by baymud** [SynthstromAudible/DelugeFirmware#3105]
### Please describe the problem:

In prior versions, you used to be able to make a large adjustment to your projects tempo then press undo to jump back up (and then redo to get back down again). For example: 

- Select a tempo of 100BPM
- Turn the tempo knob down to 80BPM
- Press undo to jump back up to 100BPM
- Press redo to drop back down to 80 BPM (this assumes you haven't done anything else for undo/redo to change)

However at some point this behaviour was lost/changed and it now works as follows: 

- Select a tempo of 100BPM
- Turn the tempo knob down to 80BPM
- Pressing undo results in 81BPM (so 20 presses to get back to 100BPM)

### What is the expected behavior?

It would be ideal to be able to resume the old behaviour of being able to undo/redo large tempo changes, not simply undoing each single BPM increment

### What hardware did you reproduce it with?

7SEG

### What firmware did you reproduce it with?

Beta

**Comment by BenUniqcode** [comment]
Yes this is annoying. Scrolling the tempo should create a single Undo entry that goes back to the value it was at when you started scrolling, rather than an entry for every 1 bpm change. The current behaviour means that more important Undo steps are lost.
